protocol/testing/version/version.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 CURRENT_VERSION=$(<VERSION_CURRENT)
 PREUPGRADE_VERSION=$(<VERSION_PREUPGRADE)
+PREUPGRADE_VERSION_FULL_NAME=$(<VERSION_FULL_NAME_PREUPGRADE)
 
```

Comment:
Do you think instead of having three files VERSION_CURRENT, VERSION_PREUPGRADE, VERSION_PREUPGRADE - it would be better to have a single file and source that file here?

Reply:
Don't think that would work too well with the embed directive I have in version.go in this directory.